modules/nlu-testing/src/bot-templates/bp-nlu-synonyms-testing/entities/airport.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "airport",
+  "name": "airport",
+  "type": "list",
+  "occurrences": [
+    { "name": "CDG", "synonyms": [] },
```

Comment:
they should all be together instead of separate, ie.

`"name": "CDG", "synonyms": [ "Paris Airport", "Charles-de-Gaules", "..." ]`
`"name": "YQB", "synonyms": [ "Quebec Airport", "Jean-Lesage", "..." ] `

Reply:
Changed "airport" to "paris airport". I agree it was confusing.